src/OrderLinesList/OrderLinesListContainer.js

```diff
@@ -107,6 +124,33 @@ const OrderLinesListContainer = ({ mutator, location }) => {
   // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [ordersMap]);
 
+  const onExportCSV = useCallback(async () => {
```

Comment:
the thing is that you're mixing up logic for order lines list and export, passing only props down for no reason. It should be a separate component , so order lines list component works on list, and new component works on export. the same for orders list